### DISCUSSION TITLE
Init as new pnpm workspace

### DISCUSSION THREAD

**Discussion by radoslavirha** [TypedProject/ts-express-decorators#2418] [Q&A] [answered]
Hello,

I'm trying to init new Ts.ED server in existing pnpm monorepo, but init process seems to be stuck.

Any ideas how to get it working? Only new `package.json` has been created.

![image](https://github.com/tsedio/tsed/assets/23722745/74f5e2e8-e06d-465a-b353-9f6ec71488bb)

Thanks!

**Comment by Romakita** [comment]
This is why the pnpm is marked as experimental 😁

**Reply by Romakita** [reply]
Add --verbose to see full logs.

**Reply by radoslavirha** [reply]
Yeah I know, but I thought init would work :)

After whole night:
![image](https://github.com/tsedio/tsed/assets/23722745/9de73c84-afb7-424b-9889-89b482ec95ee)

**Reply by Romakita** [reply, marked as answer]
Init works if you are in a fresh directory. Not in a workspace
